### ISSUE TITLE
rRNA data not carrying through to placement step

### ISSUE THREAD

**Issue by hollybik** [gjospin/PhyloSift#287]
On both master and devel branches, the rRNA data is not carrying through to the placement step, even though we get BLAST hits and subsequent alignments are produced. When using a test input file containing only confirmed 16S sequences (pulled out via QIIME), no .jplace files are produced and all Phylosift summary outputs are empty.

**Comment by hollybik** [comment]
Test files containing 16S data are on Edhar:

/home/hollybik/TestData/microBEnet/rep_set\* (4 files to pick from)

**Comment by hollybik** [comment]
This turns out to be the same issue we've been having along - you can't get placement unless you manually delete any rRNA files in the /share/phylosift/marker directory that have .updated file names. After you delete these files and re-index, everything works fine and you get .jplace files for rRNA as expected. 

So for the sake of end users, PhyloSift marker downloads shouldn't contain the .updated rRNA makers (Guillaume suggested this might be happening when we do the DBupdate to the new DNGWU markers).

**Comment by gjospin** [comment]
The "trick" to remove the .updated files from the DB worked out ok however the base 16s_rep_bac directory (same goes for 16s_rep_arc and 18s_rep) does not contain the .taxonmap file which prevents the summary from getting generated.

Checked the .updated versions of those markers and they also don't contain the .taxonmap files.

**Comment by gjospin** [comment]
Looking around for rRNA taxonmap files, I manually added them into the reference directories 
16s_rep_arc
16s_rep_bac
18s_rep

I was able to get summary files.
Also adding a condition for the decorated marker names to not add .updated to anything starting with 18s or 16s fixes the problem of having to remove the .updated 18s and 16s files/directories. I used this strategy to correctly target the .taxonmap file. 

Pushed on Devel. Closing.
